src/tooling/docs-assembler/assembler.yml

```diff
@@ -50,6 +50,134 @@ narrative:
 #   next: <git_ref>
 ###
 
+### This is not yet ready but already added
+
+#versions:
+#  stack:
+#    # This is the default versioning scheme, used for most of the docs.
+#    # It uses the stack version as the current version.
+#    base: '9.0'
+#    current: '9.0.1' # https://artifacts.elastic.co/releases/stack.json
+#    legacy_versions: [ '8.18', '8.17', '8.16', '8.15', '8.14', '8.13', '8.12', '8.11', '8.10', '8.9', '8.8', '8.7', '8.6', '8.5', '8.4', '8.3', '8.2', '8.1', '8.0', '7.17' ]
```

Comment:
I know this is here for a follow up but the `legacy_versions` can already be removed here right?

Reply:
https://github.com/elastic/docs-builder/pull/1361/commits/3d9129147029415417a035e35e47d5dc7d7c6456